fabric-resource-loader-v0/src/main/java/net/fabricmc/fabric/api/resource/ResourceManagerHelper.java

```diff
@@ -52,4 +54,27 @@ default void addReloadListener(IdentifiableResourceReloadListener listener) {
 	static ResourceManagerHelper get(ResourceType type) {
 		return ResourceManagerHelperImpl.get(type);
 	}
+
+	/**
+	 * Registers a built-in resource pack.
+	 *
+	 * <p>A built-in resource pack is an extra resource pack provided by your mod which is not always active, it's similar to the "Programmer Art" resource pack.
+	 *
+	 * <p>Why and when to use it? A built-in resource pack should be used to provide extra assets/data that should be optional with your mod but still directly provided by it.
+	 * For example it could provide textures of your mod in another resolution, or could allow to provide different styles of your assets.
+	 *
+	 * <p>The {@code subPath} corresponds to a path in the JAR file which points to the resource pack folder. For example the subPath can be {@code "resourcepacks/extra"}.
+	 *
+	 * <p>Note about the enabled by default parameter: a resource pack cannot be enabled by default, only data packs can.
```

Comment:
If this is impossible to achieve, then why is a parameter for it available? What does it actually do?

Reply:
It says it's for data packs. And note that in resource loader and yarn a "Resource Pack" is both potentially a resource pack and a data pack.

Reply:
Oh, okay, I thought this whole PR was about Resource Packs for some reason.